libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat

```diff
@@ -102,6 +102,7 @@ define HAL_BARO_BMP280_NAME "bmp280"
 define HAL_COMPASS_DEFAULT HAL_COMPASS_HMC5843
 define HAL_COMPASS_HMC5843_I2C_BUS 0
 define HAL_COMPASS_HMC5843_I2C_ADDR 0x1E
+define ALLOW_ARM_NO_COMPASS TRUE
```

Comment:
MENTAL NOTE TO OTHER DEVELOPERS that this is getting slipped in.

Reply:
its actually a correct change for boards that don't have a builtin compass

Reply:
I didn't say it wrong, it's just out of scope for this PR.

Reply:
Yes. New PR so we can get it merged faster.